demos/todomvc/src/components/App/index.js

```diff
@@ -8,7 +8,7 @@ import Recorder from '../Recorder'
 import visibleTodosRefs from '../../computed/visibleTodosRefs'
 
 export default connect({
-  todos: state`app.todos`,
+  todos: state`app.todos.**`,
```

Comment:
Would not `todos: state`app.todos.*`,` be enough as we use here only `Object.keys(props.todos).length`?